src/content/1.7/development/components/grid/tutorials/modify-grid-in-module.md

```diff
@@ -3,4 +3,170 @@ title: How to modify an existing Grid in a module?
 weight: 5
 ---
 
-> Do you know how to do it? Please feel free to contribute to the docs!
\ No newline at end of file
+{{< minver v="1.7.5.1" title="true" >}}
+
+# How to modify an existing Grid in a module?
+
+If you need to modify an existing Grid, like inverting two columns for instance, this is really easy to do it in a module thanks to the available hooks.
+
+## Business use case
+
+In this tutorial, we will try to improve the user experience of the Customers listing Grid (this page will be available in 1.7.6).
+
+Thinking about it, we could improve the current grid to highlight business-driven information:
+
+* we could remove the "Social title", not really interesting if our business is not "gender" specific,
+* we could remove the "Partner offers" column, as it doesn't bring any useful information,
+* and also remove the "Active" column, for the same reason;
+
+We could also add a new column about the number of orders per customer and re-organize a little bit all the columns.
+
+Let's see together how to do that with PrestaShop.
+
+## Removing columns and filters from a Grid
+
+Thanks to the available hooks, it's super easy to remove columns from a grid!
+
+As stated in the documentation, for each grid with the id {id} a list of hooks is available to alter the grid.
+
+First, we will use the hook able to alter the **Definition** of the grid: the Definition is a specific object where we define:
+
+* the columns,
+* the filters,
+* the grid actions,
+* and the grid bulk actions;
+
+We need to register a hook into a module (let's name it `customers` for the sake of the example).
+
+The Customers Grid id is "customer", you can find this information in the the associated class: `CustomerGridDefinitionFactory`.
+
+```php
+/**
+ * Hooks allows to modify Logs grid definition.
+ * This hook is a right place to add/remove columns or actions (bulk, grid).
+ *
+ * @param array $params
+ */
+public function hookActionCustomerGridDefinitionModifier(array $params)
```

Comment:
not sure if thats a right place, but maybe we can document what keys are available in `$params`?

Reply:
the right place would be in the "Reference" section of the documentation (imho)